kubetest/e2e.go

```diff
@@ -697,7 +704,6 @@ func kubemarkTest(testArgs []string, dump string, o options, deploy deployer) er
 			os.Environ(),
 			"KUBERNETES_PROVIDER=kubemark",
 			"KUBE_CONFIG_FILE=config-default.sh",
-			fmt.Sprintf("KUBECONFIG=%s/test/kubemark/resources/kubeconfig.kubemark", cwd),
```

Comment:
Won't this break current tests (those without cluster loader)?

Reply:
It shouldn't. `KUBECONFIG` is set 4 lines above. I assume `os.Environ()` will read it from env.